data/service/api/v1/alerts.go

```diff
@@ -24,6 +24,8 @@ func AlertsRoutes() []service.Route {
 		service.Get("/v1/users/:userId/followers/:followerUserId/alerts", GetAlert, api.RequireAuth),
 		service.Post("/v1/users/:userId/followers/:followerUserId/alerts", UpsertAlert, api.RequireAuth),
 		service.Delete("/v1/users/:userId/followers/:followerUserId/alerts", DeleteAlert, api.RequireAuth),
+		service.Get("/v1/users/:userId/followers/alerts", ListAlerts, api.RequireServer),
+		service.Get("/v1/users/without_communication", GetUsersWithoutCommunication, api.RequireServer),
```

Comment:
This API endpoint doesn't return users but `LastCommunication` and should be renamed to better reflect the resource it returns.